src/main/java/duchess/Duke.java

```diff
@@ -0,0 +1,125 @@
+package duchess;
+
+import duchess.command.Command;
+import duchess.exception.DuchessException;
+import duchess.io.Parser;
+import duchess.storage.Storage;
+import duchess.task.TaskList;
+import duchess.ui.Ui;
+
+/**
```

Comment:
I really like how you've commented throughout your codebase but especially here since this is the main `class` of the code. Kudos!! :)

Reply:
Thanks Raivat! Appreciate your help with looking through my code :)